### PR TITLE
[OneExplorer] Build a node map on demand

### PR DESCRIPTION
**Pull request by dayo09**
This commit removes the stage of pre-building a node map on the beginning. _nodeMap will collect the path-to-node relation when getTreeItem() is called, which is a function called when the node is shown on the view.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

---

From #1402

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

File: src/OneExplorer/OneExplorer.ts [hunk 1]
```diff
@@ -468,6 +468,12 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
   private fileWatcher = vscode.workspace.createFileSystemWatcher(`**/*`);
 
   private _tree: DirectoryNode|undefined;
+
+  // NOTE
+  // _nodeMap only contains the nodes which have ever been shown in ONE Explorer view by revealing
+  // the parent nodes. To get the unseen node from _nodeMap, _nodeMap needs to be pre-builted. Mind
```
Comment (llFreetimell) [inline src/OneExplorer/OneExplorer.ts:474]: Maybe? :)

```suggestion
  // the parent nodes. To get the unseen node from _nodeMap, _nodeMap needs to be pre-built. Mind
```

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**